dom.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>DOM ASSESSMENT</title>
+	<link rel="stylesheet" type="text/css" href="dom.css">
+	<script type="text/javascript" src="dom.js"></script>
+</head>
+<body>
+<header>
+	<ul class="head">
+		<ul class="head_left">
+			<li class="li"><img src="/Users/bilalshaikh/Bilal-ui-html/Dom Assessment/app_head.png" height="25x" width="25px"></li>
```

Comment:
use a tag instead of img and set its background as image url
set height and width through css